app/controllers/api/v1/reservations_controller.rb

```diff
@@ -25,6 +25,18 @@ def destroy
     render json: { status: :no_content }
   end
 
+  def mass_destroy
+    range = Range.new params[:start], params[:end]
+    reservations = Reservation.where(plane_id: params[:plane_id]).in_range(range)
+
+    reservations.each do |r|
+      r.destroy
+    end
+
+    render json: { status: :no_content }
```

Comment:
Tää meni mullakin väärin aiemmin. Pitäis olla render json: {}, status: :no_content. Muuten se vaan palauttaa json objectin jossa avaimena status ja arvona no_content